### PR TITLE
feat(DB Connection): add db connection for landing page

### PR DESCRIPTION
**Pull request by RoeiRom**

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -1,24 +1,21 @@
 import React from 'react';
+<<<<<<< HEAD
```
Comment (OrelKaravani) [inline client/src/components/App/Content/LandingPage/LandingPage.tsx]: wtf??

File: client/src/components/App/Content/LandingPage/LandingPage.tsx [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
 import {useSelector} from 'react-redux';
 import { Typography } from '@material-ui/core';
 
 import User from 'models/User';
 import StoreStateType from 'redux/storeStateType';
+=======
+>>>>>>> feat(DB Connection): add db connection for landing page
```
Comment (OrelKaravani) [inline client/src/components/App/Content/LandingPage/LandingPage.tsx]: also... wtf??

File: client/src/components/App/Content/LandingPage/LandingPageStyles.ts [hunk 3]
```diff
@@ -6,11 +6,6 @@ const useStyles = makeStyles({
     content: {
         height: '84vh',
         backgroundColor: primaryBackgroundColor,
-    },
-    welcomeMessage: {
```
Comment (OrelKaravani) [inline client/src/components/App/Content/LandingPage/LandingPageStyles.ts:10]: You have a conflict from the previous styling of that file

**Review by OrelKaravani — CHANGES_REQUESTED** [review]

**Review by OrelKaravani — APPROVED** [review]

File: client/src/commons/LoadingSpinner/LoadingSpinner.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    const isLoading = useSelector<StoreStateType, boolean>(state => state.isLoading);
+
+    const classes = useStyle();
+
+    return (
+        <>
+        {
+            isLoading &&
+                <div className={classes.container}>
+                    <CircularProgress size='20vh'/>
```
Comment (daniel7764) [inline client/src/commons/LoadingSpinner/LoadingSpinner.tsx:20]: Why inline css?

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 5]
```diff
@@ -4,47 +4,56 @@ import Table from '@material-ui/core/Table';
 import TableRow from '@material-ui/core/TableRow';
 import TableBody from '@material-ui/core/TableBody';
 import TableCell from '@material-ui/core/TableCell';
+import Typography from '@material-ui/core/Typography';
 import TableHead from '@material-ui/core/TableHead';
```
Comment (daniel7764) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:8]: Why doing the imports this way and not with {}

**Review by OrelKaravani — DISMISSED** [review]

**Review by daniel7764 — CHANGES_REQUESTED** [review]

File: client/src/commons/LoadingSpinner/LoadingSpinner.tsx (hunk 4 above)
Comment (RoeiRom) [reply client/src/commons/LoadingSpinner/LoadingSpinner.tsx:20]: it is not inline css, its the property in the component that decides the size of the spinner, the only way to control it

Comment (daniel7764) [reply client/src/commons/LoadingSpinner/LoadingSpinner.tsx:20]: Ok sorry :(

**Review by daniel7764 — APPROVED** [review]

**Review by OrelKaravani — APPROVED** [review]

**Pull request merged**